shared/oae/api/oae.api.user.js

```diff
@@ -259,6 +259,10 @@ define(['exports', 'jquery', 'underscore', 'oae.api.config'], function(exports,
                 'token': token
             },
             'success': function(data) {
+                // Accepting an invitation will set a verified email if there wasn't one already,
+                // so do that here to avoid giving the user a pop-up
+                require('oae.core').data.me.email = require('oae.core').data.me.email || data.email;
```

Comment:
In scenarios where we expect the `oae.data.me` to represent the latest version of the user profile outside of this process. Such as in `editprofile.js` to fill out the initial profile information of the form, `oae.api.js` where we determine if the user has a verified email, etc...